report_compassion/models/contract.py

```diff
@@ -51,7 +51,7 @@ def _compute_amount_due(self):
     @api.multi
     def get_gift_communication(self, product):
         self.ensure_one()
-        lang = self.partner_id.lang
+        lang = self.correspondent_id.lang
```

Comment:
In fact the gift could be either sent to the payer or the correspondent. There is a field in the contract determining this and you could use it to use the correct partner.
```suggestion
        lang = self.mapped(self.send_gifts_to).lang
```